<launch>
  
  <param name="/dimension" type="int" value="3" />
  <param name="/relaxation_rank" type="int" value="5" />
  <param name="/optimization_rate" type="double" value="1.0" />
  <param name="/pose_update_topic" type="str" value="/pose_update" />
  <param name="/num_robots" type="int" value="1" />
  <!-- Input path to the dataset -->
  <param name="/dataset" type="str" value="$(find dpgo_ros)/data/smallGrid3D.g2o" />




  <!-- Launch indivial PGO agents. Number must match /num_robots -->
  <include file="$(find dpgo_ros)/launch/PGOAgent.launch">
	  <arg name="agent_id" value="0" />
  </include>


</launch>